### ISSUE TITLE
Manifest file for Philips loader

### ISSUE THREAD

**Issue by IngmarVoigt2** [MedicalUltrasound/Image3dAPI#158]
Dear @SteveKauffman, @mdickerson110 ,

I am currently working with a version from last May (`PhilipsLoader-8May2019`) and realized users are forced to register the DLL into the Windows regsistry using adminstrator mode, since there is no manifest file provided in that version.

It would be great if a manifest file could be provided, so that an integration based on https://github.com/MedicalUltrasound/Image3dAPI/tree/master/RegFreeTest would be possible.

Who could help with this? Is such an updated package already available?

Kind regards

**Comment by lassoan** [comment]
We work a lot with Philips 4D ultrasound images and so far we have been limited to QLab-exported Cartesian images, but lack of ECG, Doppler, etc. hurts us. Who can we get the Philips loader from?

**Comment by mattjolley** [comment]
Yes.  We are very interested in this and work with @lassoan.  Please let us know what we could do to help improve access to the full 3D US data (color, standard, EKG, full resolution).

**Comment by lassoan** [comment]
@IngmarVoigt2 Could you help us? Which person/group within Philips you got `PhilipsLoader-8May2019` from?

**Comment by IngmarVoigt2** [comment]
unfortunately only indirectly via internal contacts - @forderud can you point them to the right people?

**Comment by SteveKauffman** [comment]
It appears that I ignored this query when it initially occurred in March.  I should have made sure it got properly handed over to Scott Settlemier, who’s covering this project for us now.  Looking at it now, and will get you a response soon.

--Steve

**Comment by tbirdso** [comment]
Hi all, I am interested in learning more on the status of the Philips Image3DAPI loader for Philips 3D/4D US images. Is that library still maintained? If so, how are releases made available, and who should I talk to in order to learn more? Is Scott Settlemier still leading that effort at Philips?

@lassoan or @mattjolley , would you be able to comment on whether you were previously able to obtain the `PhilipsLoader-8May2019` DLL, in addition to the Philips export steps you've [documented in SlicerHeart](https://github.com/SlicerHeart/SlicerHeart/blob/master/Docs/ImageImportExport.md#import-philips-4d-cardiac-images)?

Thank you for your help.

**Comment by mattjolley** [comment]
@tbirdso 

We have not obtained the Philips dll.  Last we checked Philips had not released that publically and the library seemed intended for intervendor work, but we hope that has evolved.

We have not checked in with the Philips team on this in a while, so it would be good to do so.

Scott Settlemier was still on that team the last time we connected.

We have used the GE equivalent and that is pretty easy to obtain from GE.

**Comment by tbirdso** [comment]
@mattjolley thank you for the update! Disappointing to hear that Philips did not release their library at the time. It does seem that GE is the current leader in this regard.

I am unclear on whether Scott Settlemier is active on GitHub. Perhaps someone on the Philips team would be willing to point me in the right direction? @SteveKauffman , @mdickerson110 ?

**Comment by cacoool** [comment]
Hello everyone,

We're interested in using the tool for Philips 3/4D US, as an alternative to qlab, which we find cumbersome. I was wondering if there have been any recent updates or developments with the Philips Image3DAPI loader since the previous discussion last year.

**Comment by lassoan** [comment]
Accessing the 3D/4D ultrasound images that you acquire is a very basic need and I even consider unethical if not illegal that several vendors deny patients and researchers open access to this data. Unfortunately, it seems that the only practically feasible way to enforce vendors to change their behavior is to not buy systems that restrict access to your data.

So, if you are (as many of us) unhappy with how some imaging vendors do not let you access your data, next time your hospital purchases new systems, make sure that "open access to 3D/4D imaging data" is among the requirements and do not let them get away with limited workarounds.
